tools/wpt/run.py

```diff
@@ -67,10 +67,14 @@ def exit(msg=None):
 
 
 def args_general(kwargs):
-    kwargs.set_if_none("tests_root", wpt_root)
-    kwargs.set_if_none("metadata_root", wpt_root)
-    kwargs.set_if_none("manifest_update", True)
-    kwargs.set_if_none("manifest_download", True)
+    def set_if_none(name, value):
```

Comment:
Blank lines before and after the closure.

Reply:
Done!